.github/renovate.json5

```diff
@@ -50,6 +51,10 @@
         "matchPackageNames": ["offchainlabs/arb-node", "offchainlabs/nitro-node"],
         "versioning": "regex:^v?(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)(-(?<build>[a-zA-Z0-9]+))?$"
       },
+      {
+        "matchPackagePatterns": ["avaplatform/avalanchego"],
```

Comment:
matchPackagePatterns is going to only match the avalanche updates, not say.. some Chart dependency (like init-container images) that also are liable to get an update PR as they do sometimes.
We can leave the  "Avalanche Group" maybe, as it was, and just add the enabled: false bit to it?